packages/expo-cli/src/commands/upgradeAsync.ts

```diff
@@ -181,7 +181,7 @@ export function getDependenciesFromBundledNativeModules({
   }
 
   // Get the supported react/react-native/react-dom versions and other related packages
-  if (workflow === 'managed' || projectDependencies['expokit']) {
+  if (projectDependencies['expokit']) {
```

Comment:
we should keep using the expoReactNativeTag for sdk <= 42

Reply:
good catch! the pr is updated accordingly, please take a look. thanks!